src/pie-chart/pie-grid.component.ts

```diff
@@ -104,6 +104,8 @@ export class PieGridComponent extends BaseChartComponent {
   @Input() tooltipText: (o: any) => any;
   @Input() label: string = 'Total';
   @Input() minWidth: number = 150;
+  @Input() hidePercentage: boolean = false;
+  @Input() hideSeriesLabel: boolean = false;
 
```

Comment:
I would like suggest you to add separate variables for Dy, instead of using methods in template
```suggestion
percentageDy = '-0.5em';
seriesLabelDy = '0.5em';
```

Reply:
Also here we need to add 'implements OnInit' to the component declaration
```suggestion
  ngOnInit(): void {
    if (this.hidePercentage && !this.hideSeriesLabel) {
      this.percentageDy = '0.0em';
      this.seriesLabelDy = '0.0em';
    }
  }
```